src/main/java/com/example/onlinebookstore/mapper/BookMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.example.onlinebookstore.mapper;
+
+import com.example.onlinebookstore.dto.BookDto;
+import com.example.onlinebookstore.dto.CreateBookRequestDto;
+import com.example.onlinebookstore.model.Book;
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.NullValueCheckStrategy;
+
+@Mapper(
+        componentModel = "spring",
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+        nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
+        implementationPackage = "<PACKAGE_NAME>.impl")
+public interface BookMapper {
+    BookDto toDto(Book book);
+
+    Book toBook(CreateBookRequestDto createBookRequestDto);
+
+    void updateBookFromBookDto(BookDto bookDto, @MappingTarget Book book);
```

Comment:
minor

may be simplified:
```suggestion
    void updateBook(BookDto bookDto, @MappingTarget Book book);
```